tools/actions/has-hash-commit-deps/build/index.js

```diff
@@ -1 +1 @@
this;TunnelingAgent.prototype.createSocket.call(r,e,(function(n){var o=e.request.getHeader("host");var i=mergeOptions({},r.options,{socket:n,servername:o?o.replace(/:.*$/,""):e.host});var a=s.connect(0,i);r.sockets[r.sockets.indexOf(n)]=a;t(a)}))}function toOptions(e,t,r){if(typeof e==="string"){return{host:e,port:t,localAddress:r}}return e}function mergeOptions(e){for(var t=1,r=arguments.length;t<r;++t){var n=arguments[t];if(typeof n==="object"){var s=Object.keys(n);for(var o=0,i=s.length;o<i;++o){var a=s[o];if(n[a]!==undefined){e[a]=n[a]}}}}return e}var l;if(process.env.NODE_DEBUG&&/\btunnel\b/.test(process.env.NODE_DEBUG)){l=function(){var e=Array.prototype.slice.call(arguments);if(typeof e[0]==="string"){e[0]="TUNNEL: "+e[0]}else{e.unshift("TUNNEL:")}console.error.apply(console,e)}}else{l=function(){}}t.debug=l},491:e=>{"use strict";e.exports=require("assert")},361:e=>{"use strict";e.exports=require("events")},147:e=>{"use strict";e.exports=require("fs")},685:e=>{"use strict";e.exports=require("http")},687:e=>{"use strict";e.exports=require("https")},808:e=>{"use strict";e.exports=require("net")},37:e=>{"use strict";e.exports=require("os")},17:e=>{"use strict";e.exports=require("path")},404:e=>{"use strict";e.exports=require("tls")},837:e=>{"use strict";e.exports=require("util")}};var t={};function __nccwpck_require__(r){var n=t[r];if(n!==undefined){return n.exports}var s=t[r]={exports:{}};var o=true;try{e[r].call(s.exports,s,s.exports,__nccwpck_require__);o=false}finally{if(o)delete t[r]}return s.exports}(()=>{__nccwpck_require__.n=e=>{var t=e&&e.__esModule?()=>e["default"]:()=>e;__nccwpck_require__.d(t,{a:t});return t}})();(()=>{__nccwpck_require__.d=(e,t)=>{for(var r in t){if(__nccwpck_require__.o(t,r)&&!__nccwpck_require__.o(e,r)){Object.defineProperty(e,r,{enumerable:true,get:t[r]})}}}})();(()=>{__nccwpck_require__.o=(e,t)=>Object.prototype.hasOwnProperty.call(e,t)})();(()=>{__nccwpck_require__.r=e=>{if(typeof Symbol!=="undefined"&&Symbol.toStringTag){Object.defineProperty(e,Symbol.toStringTag,{value:"Module"})}Object.defineProperty(e,"__esModule",{value:true})}})();if(typeof __nccwpck_require__!=="undefined")__nccwpck_require__.ab=__dirname+"/";var r={};(()=>{"use strict";__nccwpck_require__.r(r);var e=__nccwpck_require__(352);var t=__nccwpck_require__.n(e);var n=__nccwpck_require__(17);var s=__nccwpck_require__.n(n);var o=__nccwpck_require__(147);var i=__nccwpck_require__.n(o);async function main(){const t=(0,n.join)(e.getInput("workspace")||"","package.json");e.info(t);if((0,o.existsSync)(t)){try{const r=await o.promises.readFile(t,"utf8");e.info(r);if(!r){e.setOutput("has-hash-commit-deps",false);return}const n=JSON.parse(r);const{dependencies:s={},devDependencies:i={}}=n;const a=Object.keys(s).map((e=>s[e])).concat(Object.keys(i).map((e=>i[e])));const u=a.some((e=>e.includes(".git")||e.includes("#")));e.setOutput("has-hash-commit-deps",u);return}catch(t){e.info(t);throw t}}}main().catch((t=>e.setFailed(t)))})();module.exports=r})();
```

Comment:
Not completely sure why this file has changed ?

Reply:
@valpinkman looks like this file gets updated when one runs `pnpm build`. Don't know if it's expected but we might want to update this built file in a separated PR (or add the build folder to a `.gitignore` if it is not supposed to be updated anymore)
In the meantime, I reverted this change in this PR 👍